src/components/NewTask.tsx

```diff
@@ -1,27 +1,34 @@
 import { ChangeEvent } from 'react';
-import styles from '../App.module.scss';
-import Classnames from 'classnames';
-import { Button as SemanticButton} from 'semantic-ui-react'
-import 'semantic-ui-css/semantic.min.css'
+import styles from './NewTask.module.scss';
+import { Button as SemanticButton } from 'semantic-ui-react';
 
-function NewTask( props: {item: string, onAdd: () => void, onClear: () => void, typeText: (event: ChangeEvent<HTMLInputElement>) => void}) {
+function NewTask(props: { item: string; onAdd: () => void; onClear: () => void; typeText: (event: ChangeEvent<HTMLInputElement>) => void }) {
```

Comment:
`typeText` is not a meaningful name, the function it represents is an input's `onChange` event handler, so as mentioned previously `handleChange`.

Reply:
Done.